source/developer-guides/configuration.rst

```diff
@@ -203,6 +203,10 @@ folder.
 Run this Python\* script in the command window: **CfgDataStitch.py -h**
 for parameters for this script.
 
+Example:
+python CfgDataStitch.py -i sbl_ifwi_old.bin -k ConfigTestKey_Priv_RSA3072.pem -s SblOpen\BootloaderCorePkg\Tools -c .

```

Comment:
When providing command examples, can you use the format specifier (::)? Please refer to other examples in the same file.

Reply:
Got it